<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCommandDialog</class>
 <widget class="QDialog" name="EditCommandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="commandTitleLabel">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="commandTitleLineEdit">
           <property name="text">
            <string>AT+&lt;&gt; somth</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="commandLabel">
           <property name="text">
            <string>Command</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="commandLineEdit">
           <property name="text">
            <string>AT+&lt;&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="sendPushButton">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="InfoLabel">
       <property name="text">
        <string>Info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="commandInfoPlainTextEdit">
       <property name="plainText">
        <string>Any info about command</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
